### PR TITLE
Fixes #32168 - consume repositories from yum.theforeman.org

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/katello/katello-repos/katello.repo [hunk 1]
```diff
@@ -31,14 +31,14 @@ module_hotfixes=1
 
 [katello-source]
 name=Katello @REPO_NAME@ Source
-baseurl=https://fedorapeople.org/groups/katello/releases/source/srpm/@REPO_VERSION@/katello/@DIST@/
+baseurl=https://yum.theforeman.org/katello@REPO_VERSION@/katello/@DIST@/source/
```
Comment (ekohl) [inline packages/katello/katello-repos/katello.repo:34]:
```suggestion
baseurl=https://yum.theforeman.org/katello/@REPO_VERSION@/katello/@DIST@/source/
```

File: packages/katello/katello-repos/katello.repo [hunk 2]
```diff
@@ -31,14 +31,14 @@ module_hotfixes=1
...
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
 enabled=0
 gpgcheck=@REPO_GPGCHECK@
 
 [katello-candlepin-source]
 name=Katello Candlepin source
-baseurl=https://fedorapeople.org/groups/katello/releases/source/srpm/@REPO_VERSION@/candlepin/@DIST@/
+baseurl=https://yum.theforeman.org/katello@REPO_VERSION@/candlepin/@DIST@/source/
```
Comment (ekohl) [inline packages/katello/katello-repos/katello.repo:41]:
```suggestion
baseurl=https://yum.theforeman.org/katello/@REPO_VERSION@/candlepin/@DIST@/source/
```

File: packages/katello/katello-repos/katello.repo (hunk 1 above)
Comment (evgeni) [reply packages/katello/katello-repos/katello.repo:34]: DAMN YOU SED!

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Perhaps this needs a Redmine issue?

**Review by ekohl — APPROVED** [review]

**Pull request merged**